src/tiled/abstractobjecttool.cpp

```diff
@@ -115,8 +116,7 @@ void AbstractObjectTool::mouseMoved(const QPointF &pos,
 void AbstractObjectTool::mousePressed(QGraphicsSceneMouseEvent *event)
 {
     if (event->button() == Qt::RightButton) {
-        showContextMenu(topMostObjectItemAt(event->scenePos()),
-                        event->screenPos());
+        showContextMenu(event->scenePos(), event->screenPos());
```

Comment:
The line with showContextMenu. Because the change from `topMostObjectItemAt` to `event->scenePos() `is no longer required.